src/router/index.js

```diff
@@ -0,0 +1,22 @@
+import VueRouter from "vue-router";
+
+import LogInPage from "@/views/LogInPage.vue";
+
+const routes = [
+  {
+    path: "/",
+    name: "LogIn",
+    component: LogInPage,
```

Comment:
Right now this definition will make it so that we have a log in page as our "home" page.
You could import and use your HomePage component here instead.
Also, feel free to import and define routing for other pages you created too! 😄